src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/providers/NotcreatedDBInfluxDB.java

```diff
@@ -0,0 +1,230 @@
+//            System.out.println("Tables in Restore Db db_1  ----" + restoredbtble);
+
+            for (int i = 0; i < restoredbtble.size(); i++) {
+                List<Object> restoretble = restoredbtble.get(i);
+                for (Object restoretable : restoretble) {
+                    //QueryResult rowcount = influxDB.query(new Query("SELECT count(*) FROM /.*/" , "db_1"));
+                    QueryResult rowcount = influxDB.query(new Query("SELECT count(*) FROM "+ restoretable  , "db_1"));
+                    List<QueryResult.Result> restoretbleresults =rowcount.getResults();
+                    if(restoretbleresults.get(0).hasError()) {
+                    }else{
+                        List<List<Object>> resdbtblecnt = restoretbleresults.get(0).getSeries().get(0).getValues();
+                        //  System.out.println("RowCount in restore db table  "  + resdbtblecnt);
+
+                        // Get the row count of each table from the restore database
+
+                        for (int j = 0; j < resdbtblecnt.size(); j++) {
+                            List<Object> resrowcount = resdbtblecnt.get(j);
+                            //   System.out.println("Row count from each table in restore db " + resrowcount.get(1));
+                            resrowcountval = Float.valueOf(resrowcount.get(1).toString());
+                            resrwcnt.add(Float.valueOf(resrowcount.get(1).toString()));
+                            //System.out.println("Row count from each table in restore db " + resrowcountval);
+
+                        }
+
+                    }
+                    restblecount = i+1;
+                }
+            }
+
+//            System.out.println("Total Table in restore db  "  + restblecount);
+
+            // Compare total db count in backup db and restore db
+
+            if(bkdbcount == resdbcount) {
+//                System.out.println("DB count equal in Backup and Restore DB");
+            }else
+            {
+//                System.out.println("DB count not equal in Backup and Restore DB");
+            }
+
+            // Compare total table count in backup db and restore db
+
+            if(bktblecount == restblecount) {
+                //   System.out.println("Table count equal in Backup and Restore DB");
+            }else
+            {
+                // System.out.println("Table count not equal in Backup and Restore DB");
+            }
+
+            // Compare total rowcount in backup db and restore db
+            if(bkrwcnt.equals(resrwcnt)){
+                //       System.out.println("Rowcount in all the tables are equal!!!!");
+            }
+            else{
+                //     System.out.println("Rowcount are not correct!!!!");
+            }
+
+        }
+        catch (Exception e){
+            if(e.equals(null)) {
+                System.out.println("False");
+            }
+            e.printStackTrace();
+        }
```

Comment:
I can't tell what the benefit is for all this code. The println statements are all commented out and even then an automated verification shouldn't rely on printed output, but instead should programmatically compare and raise exceptions or log errors if comparisons fail.